catkit/hardware/iris_ao/segmented_dm_command.py

```diff
@@ -10,7 +10,7 @@
                                                      wavelength,
                                                      testbed_config_id,
                                                      apply_flat_map=True,
-                                                     filename_flat=FLAT_COMMAND_FILE)
+                                                     filename_flat='repo-path/hardware/iris-ao/CustomFLAT.ini')
```

Comment:
[revision] ``None`` or ``''``.

Reply:
@kjbrooks suggested I just put an example in here, since it's a comment to help guide the user.

Reply:
Then do so in the docstring, not the actual default arg.

Reply:
it's not an arg though, it's a docstring

Reply:
Hahahaha, I hadn't noticed that. Forget I said anything - these are not the droids you're looking for.